tests/src/Simple/Interfaces/Interfaces.cs

```diff
@@ -234,4 +237,52 @@ public string GetAString()
         }
     }
     #endregion
+
+    #region Array Interfaces Test
+    private static int TestArrayInterfaces()
+    {
+        {
+            object stringArray = new string[] { "A", "B", "C", "D" };
+
+            Console.WriteLine("Testing IEnumerable<T> on array...");
+            string result = String.Empty;
+            foreach (var s in (System.Collections.Generic.IEnumerable<string>)stringArray)
+                result += s;
+
+            if (result != "ABCD")
+            {
+                Console.WriteLine("Failed.");
+                return Fail;
+            }
+        }
+
```

Comment:
Noted, but since we don't emit variance information on EETypes yet, I don't think this will work.